setup/ci/wercker.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: single
+title:  "Wercker"
+sidebar:
+  nav: setup
+---
+
+{% include toc %}
+
+[Wercker](http://www.wercker.com) is a cloud CI system, which can be 
+enabled within Spinnaker as a pipeline trigger and also as a pipeline stage. The setup required
+is described below.
+
+## Links
+[Main Wercker Site](https://app.wercker.com)
+
+[Wercker Documentation](https://devcenter.wercker.com/)
+
+## Prerequisites
+
+To enable Wercker integration in Spinnaker, you will need to have:
+1. A login to Wercker, which you can set up at https://app.wercker.com/
```

Comment:
Suggest changing these from numbers to bullets.

Reply:
done